### PR TITLE
Adding operation name to LLM spans 

### PR DESCRIPTION
**Pull request by sudivate**
Fixes #

## Changes

Adding operation name to LLM spans 

Note: if the PR is touching an area that is not listed in the [existing areas](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md), or the area does not have sufficient [domain experts coverage](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/CODEOWNERS), the PR might be tagged as [experts needed](https://github.com/open-telemetry/semantic-conventions/labels/experts%20needed) and move slowly until experts are identified.

## Merge requirement checklist

* [ ] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: docs/gen-ai/llm-spans.md [hunk 1]
```diff
@@ -39,6 +40,19 @@ These attributes track input data and metadata for a request to an LLM. Each att
 <!-- semconv gen_ai.request -->
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
+<<<<<<< HEAD
```
Comment (karthikscale3) [inline docs/gen-ai/llm-spans.md:43]: nit: accidental string

File: model/registry/gen-ai.yaml [hunk 2]
```diff
@@ -47,6 +47,12 @@ groups:
         brief: The unique identifier for the completion.
         examples: ['chatcmpl-123']
         tag: llm-generic-response
+      - id: operation.name
+        stability: experimental
+        type: string
```
Comment (nirga) [inline model/registry/gen-ai.yaml:52]: It's better to have an `enum` here and list the options below

File: model/registry/gen-ai.yaml [hunk 3]
```diff
@@ -47,6 +47,12 @@ groups:
...
+        brief: The name of the LLM operation request being made.
+        examples: ['chat.completions', 'embeddings', 'speech.generations', 'audio.transcriptions', 'audio.translations', 'image.generations']
```
Comment (nirga) [inline model/registry/gen-ai.yaml:54]:
```suggestion
        examples: ['chat', 'embeddings', 'speech.generations', 'audio.transcriptions', 'audio.translations', 'image.generations']
```

**Review by lmolkova — COMMENTED** [review]
Please add the [changelog](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#4-changelog) record

File: model/registry/gen-ai.yaml (hunk 3 above)
Comment (sudivate) [reply model/registry/gen-ai.yaml]: True, ideally it should be completions and chat.completions

File: docs/gen-ai/llm-spans.md [hunk 4]
```diff
@@ -10,9 +10,10 @@ linkTitle: LLM requests
 
 <!-- toc -->
 
-- [Configuration](#configuration)
-- [LLM Request attributes](#llm-request-attributes)
-- [Events](#events)
+- [Semantic Conventions for LLM requests](#semantic-conventions-for-llm-requests)
```
Comment (trisch-me) [inline docs/gen-ai/llm-spans.md:13]: is toc correct here? I don't see any changes in the file itself

**Review by karthikscale3 — COMMENTED** [review]
The changes look good to me. Thanks

File: model/registry/gen-ai.yaml (hunk 3 above)
Comment (nirga) [reply model/registry/gen-ai.yaml]: I find `chat.completions` to be highly confusing with `completions`. I'd stick with `chat` and `completions`

File: docs/attributes-registry/gen-ai.md [hunk 5]
```diff
@@ -28,6 +29,18 @@ This document defines the attributes used to describe telemetry in the context o
 **[1]:** It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
 **[2]:** It's RECOMMENDED to format prompts as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
 
+`gen_ai.operation.name` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value                  | Description          | Stability                                                        |
+| ---------------------- | -------------------- | ---------------------------------------------------------------- |
+| `completions`          | Completions          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```
Comment (gyliu513) [inline docs/attributes-registry/gen-ai.md:36]: Howa about adding `tokenize` as well?

tokenize(prompt, return_tokens=False)[[source]](https://ibm.github.io/watson-machine-learning-sdk/_modules/ibm_watson_machine_learning/foundation_models/model.html#Model.tokenize)
The text tokenize operation allows you to check the conversion of provided input to tokens for a given model. It splits text into words or sub-words, which then are converted to ids through a look-up table (vocabulary). Tokenization allows the model to have a reasonable vocabulary size.

**Pull request closed without merging**